### DISCUSSION TITLE
Support multi spaces in stat values

### DISCUSSION THREAD

**Discussion by vopani** [h2oai/wave#1665] [Q&A]
How do I use a text with multiple spaces in `ui.stat`?

I am trying:
ui.stat(label='Category 1 &nbsp; &nbsp;x')
but it always renders with a single space only between the `1` and `x`.
<img width="81" alt="Screenshot 2022-10-10 at 3 51 28 PM" src="https://user-images.githubusercontent.com/8954217/194845212-a54e98f0-d7dd-4355-aafe-b118a829298f.png">

**Comment by mturoci** [comment]
This is the default HTML behavior - multiple spaces are merged into one. There are ways to preserve whitespace if significant via CSS so we can add it if you really need it.

**Reply by vopani** [reply]
It's not a big requirement, just for some beautification but HTML supports using `&nbsp;` for extra spaces that doesn't work when used in `ui.stat`. I was at least expecting this alternative to work if you say the component replicates HTML behaviour.

Even here on GH I can use `&nbsp;`:
`Sample &nbsp;&nbsp;&nbsp; text` : Sample &nbsp;&nbsp;&nbsp; text

**Reply by mturoci** [reply]
That's fair, I would assume the same. Can you please file a feature request?
